ui/raidboss/data/06-ew/raid/p6s.ts

```diff
@@ -69,6 +70,25 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P6S Exocleaver Healer Groups',
+      // Unholy Darkness stack headmarkers are same time as first Exocleaver
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['7869', '786B'], source: 'Hegemone', capture: false }),
+      condition: (data) => !data.secondExocleavers,
+      alertText: (_data, _matches, output) => output.healerGroups!(),
+      run: (data) => data.secondExocleavers = true,
+      outputStrings: {
+        healerGroups: Outputs.healerGroups,
+      },
+    },
+    {
+      id: 'P6S Exocleaver Move',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['7869', '786B'], source: 'Hegemone' }),
+      delaySeconds: (_data, matches) => parseFloat(matches.castTime),
```

Comment:
`castTime` in the log line is 0.3s shorter than the actual cast time.  Do you think it'd be safer to use `Ability` instead of `StartsUsing` with a `suppressSeconds`?

Reply:
I think it snapshots on the cast, but it doesn't hurt to change. Also looking at some logs, it might always be 786B + 786A x6 followed by 786B x6, so might be possible to update timeline and remove the 7869 trigger match and just use 786B without supressSeconds.

Reply:
Yeah, I'm just saying that `parseFloat(matches.castTime)` is not the end of the cast, it's 0.3s before the end of the cast.  If you look at a log file with a cast and an ability usage, the cast time is always 0.3s shorter than when the ability shows up.